### PR TITLE
added classes to sections and padding in css

### PR DESCRIPTION
**Pull request by cattix**
Check out updates to sections and new padding.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,9 +87,9 @@ <h1 class="nav-main__logo">
     </div>
   </section>
 
-  <section>
+  <section class="foot-main__section">
```
Comment (ivstudio) [inline index.html]: its footer not foot

File: index.html [hunk 2]
```diff
@@ -5,14 +5,15 @@
...
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   <link rel="stylesheet" href="./css/main.css"/>
+  <link rel="stylesheet" href="./css/images.css"/>
 </head>
 
 
 <body>
 
-  <section>
+  <section class="main-nav__section">
```
Comment (ivstudio) [inline index.html]: Be consistent with naming conventions  - nav-main__section

**Comment by cattix** [comment]
made revisions

**Comment by cattix** [comment]
will open new pr

**Pull request closed without merging**